task_1/sergii.garanenko/src/main/java/Config.java

```diff
@@ -0,0 +1,18 @@
+import org.springframework.beans.factory.config.PropertyPlaceholderConfigurer;
+import org.springframework.boot.SpringBootConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.core.io.ClassPathResource;
+
+@SpringBootConfiguration
```

Comment:
Consider that in real world someone can find using spring an overhead in every task, while for someone it can be perfectly legal.